setup.py

```diff
@@ -82,7 +82,7 @@
     'invenio-records-rest[datacite]==1.7.2',
     'invenio-records-ui==1.2.0',
     'invenio-records==1.4.0a3',
-    'invenio-search-ui==2.0.1',
+    'invenio-search-ui==2.0.4',
```

Comment:
Thanks @ntarocco for the tip about [semantic-ui-react/css-patch](https://github.com/inveniosoftware/invenio-search-ui/blob/master/invenio_search_ui/webpack.py#L33). The upgrade fixes the problem nicely.